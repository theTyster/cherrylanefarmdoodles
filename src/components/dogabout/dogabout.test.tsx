import { describe, test, expect } from "vitest";
import DogAbout from "@/components/dogabout/dogabout";
import React from 'react';

describe("DogAbout Component", () => {
    const dogData = {
      adultName: "Dog Name",
      about: "This is a dog",
    };
  test("DogAbout component should be defined", () => {
    expect.soft(<DogAbout dogData={dogData} />).toBeTruthy();
  });
  test("data passed into DogAbout component should be consistent", () => {
    expect.soft(<DogAbout dogData={dogData} />).toMatchSnapshot();
  
  })
});
