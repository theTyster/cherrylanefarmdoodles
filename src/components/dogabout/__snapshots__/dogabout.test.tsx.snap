// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`DogAbout Component > data passed into DogAbout component should be consistent 1`] = `
<DogAbout
  dogData={
    {
      "about": "This is a dog",
      "adultName": "Dog Name",
    }
  }
/>
`;
